### ISSUE TITLE
Designer render not correct after reload

### ISSUE THREAD

**Issue by ptmcode** [totaljs/flow#105]
I tried to test by creating basic test of 1 flow adding 3 elements and connecting them together. After saved, when I closed the designer and reopen it, everything still same but after I reload the project and reopen the flow in designer, it shows one missing element. 

I am doubt that it is being used in production by many.

**Comment by petersirka** [comment]
Hi @ptmcode,
Have you applied your Flow design? Only that "Apply" button saves and processes your Flow design.

![20230603-203630](https://github.com/totaljs/flow/assets/2414252/78a1f9c5-f9fe-48d0-8e2b-f5564d0b89eb)

**Comment by ptmcode** [comment]
Yes, I already apply. It is same as save changes. After clicked Apply, I closed the flow (as in your screenshot it is Total DB) and reopen I can see the latest change but when I saved change the project to make it reload and open the flow again, it missed what have changed previously.

I tried to store the change to the database and restore it back. I can see the change was there but the issue is only when reload and reopen in the flow designer.

**Comment by petersirka** [comment]
Can you make an example? Please make a zip file with Flow + node_modules inside + try to create a simple Flow --> Trigger + Print and send me the source code.

**Comment by ptmcode** [comment]
I am preparing it will send to you when done.

**Comment by ptmcode** [comment]
Please find attached file is my testing project. It is clone from github with nothing change. You can try open the flow, add more element, connecting them. Then save change, reload the project and open the flow. Also try to add Merge component because mostly it is not showing.

[flow-master.zip](https://github.com/totaljs/flow/files/11643467/flow-master.zip)

**Comment by ptmcode** [comment]
To make sure the project reload, every time I click Apply, it will show like the last 4 lines:

<img width="428" alt="image" src="https://github.com/totaljs/flow/assets/6446571/4d7c2afe-18e5-48a4-b210-6e24afbdc349">

**Comment by petersirka** [comment]
I found a problem with loading of the flow schema (I must look into this problem in more detail). Can you install Total4 beta and test it again on your side? I hope my fix will solve your problem.

`$ npm install total4@beta`.

**Comment by petersirka** [comment]
@ptmcode I found a bug and it's fixed. Please download the Flow app again. I improved "something" in the Total.js framework and found a bug in the flowstream module in this repository.

Thank you!

**Comment by ptmcode** [comment]
Let me try again, thank you.

**Comment by petersirka** [comment]
Please let me know if it works.

**Comment by ptmcode** [comment]
It is ok now, thank you.

**Comment by ptmcode** [comment]
Do you have 2 below components?

1- Flow block: it is a component that contains other flows which I can double click to see other flows inside it.
2- Flow block component: it is a component that contains other flows and cannot change during design. But it can be reused with other flow which change once affect to all.
